Rakefile

```diff
@@ -10,5 +10,5 @@ rescue LoadError
 end
 
 task :dependaseal do
-  Dependaseal.new.execute
+  Dependaseal.new.send_simple_message
```

Comment:
Will need 2 rake tasks, but will add that when I add the full slack message functionality